### DISCUSSION TITLE
Bowflex IC Bike SE 0 RPM/0 Watts

### DISCUSSION THREAD

**Discussion by K-GrooveEnjoyer** [doudar/SmartSpin2k#598] [Q&A]
I installed an SS2K on my IC bike SE, it seems to connect to the bike but I'm not getting anything on the resistance chart from it. I looked in past tickets and it seems like support was added, I even tried downgrading to the old "fixed" firmware but no changes.

**Comment by K-GrooveEnjoyer** [comment]
I have downgraded to 24.2.16 firmware with the littlefs.bin from the latest firmware release and it's reporting power now, will stick with this at the moment.

**Comment by liuwh08** [comment]
I have the same issue. Thanks for pointing out 24.2.16 works. Looks like a regression?

**Reply by K-GrooveEnjoyer** [reply]
It's kind of half broken on 24.2.16. You can't manually pair it, you just have to let it auto pair. What I have to do is let the bike power on, then afterward plug in the smartspin and it will pair with the bike and my heart rate monitor.

**Reply by liuwh08** [reply]
For me, smartspin2k could connect and report pwr and cadence but the LCD screen only shows app connect without any other information like resistance.

**Reply by eMadman** [reply]
@K-GrooveEnjoyer , @liuwh08  - could I bug one of you to [record a log](https://github.com/doudar/SmartSpin2k/wiki/Viewing-logs-via-UDP) and report a new issue for this?  The log might help identify what's happening in the background.

**Reply by liuwh08** [reply]
Enclosed are the logs for head release and 24.2.16. I kept the bike on all the time.

Looks like the latest one just connect for a short amount of time then it has to reconnect again.

[24.2.16.txt](https://github.com/user-attachments/files/18115343/24.2.16.txt)

[24.12.8.txt](https://github.com/user-attachments/files/18115375/24.12.8.txt)

**Comment by liuwh08** [comment]
Opened https://github.com/doudar/SmartSpin2k/issues/605. I guess we could move the discussion there.
